core/src/messages/dns.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{
+    nats::{SubscribeSubject, TypedMessage},
+    types::ClusterName,
+};
+use serde::{Deserialize, Serialize};
+use trust_dns_server::client::rr::RecordType;
+
+#[derive(Serialize, Deserialize, Debug)]
+pub enum LocalDnsRecordType {
+    A,
+    TXT,
+    AAAA,
+    Unknown(u16),
```

Comment:
For other record types, I think we can just make them unrepresentable. We don't need this to be a full DNS server, just enough to route traffic and get certs.

Reply:
So just have those three in the enum? Sounds good.

Reply:
Yep exactly.